iron-list.html

```diff
@@ -1255,6 +1255,13 @@
         var item = this.items && this.items[vidx];
         if (item != null) {
           inst[this.as] = item;
+          // Grab all top level keys for our item and trigger updates to the children.
+          var keys = Object.keys(item);
+          for (var i = 0; i < keys.length; i++) {
```

Comment:
I will not want to introduce this cost here. Btw, I think this issue should be fixed in 2.0. Would you be willing to change this PR, so that you delete the DOM instances instead of setting `[hidden]` on them?